.github/workflows/android.yml

```diff
@@ -24,10 +24,8 @@ jobs:
         # https://medium.com/@ychescale9/running-android-instrumented-tests-on-ci-from-bitrise-io-to-github-actions-a611e57ce71f
         key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}
         restore-keys: ${{ runner.os }}-gradle-
-    - name: Build with Gradle
-      run: ./gradlew build
-    - name: Run Unit tests
-      run: ./gradlew test --stacktrace
+    - name: Run Checks
```

Comment:
I don't think detekt was running before. "check" builds all modules, runs all tests, and runs detekt and other linters.

Reply:
I think it definitely was as I've had to fix multiple detekt errors in other PR's. Might be from some other task but it was running for sure